design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -257,6 +258,50 @@ VMI status and in one case to pass the data (processed) to the
 
 - DownwardAPI
 
+##### Device Info
+
+In some cases the multus status annotation
+contains [device-info](https://github.com/k8snetworkplumbingwg/device-info-spec/blob/main/SPEC.md).
+This info may be valuable for the binding plugin sidecar. For example
+`SRIOV` and `VDPA` plugins need this info.
+
+As the multus status annotation is not there yet on `virt-launcher` pod
+creation, the suggested solution is to use downwardAPI volume and mount
+it to the sidecar.
+The downwardAPI volume will track a new `virt-launcher` pod
+annotation `kubevirt.io/network-info` that will contain the `network-info`
```

Comment:
Changing it to network-info immediately raise question "why passing network info and not any other info?"
I would keep the scope narrow as before and have the annotation provide device-info only.

It seems that other network info already passed to the virt-launcher app through other channels (VMI status/gRPC, guest-agent), the only missing one was the device-info that up until the SR-IOV mapping bug would be given by the device-plugin.

Reply:
The reason I changed it is that in the CR we define - `DownwardAPI: device-info`. Meaning the API is extendable and we may add other DownwardAPI type. I want the CR API to be align to the pod one.